src/components/ONSPanel.tsx

```diff
@@ -17,13 +17,13 @@ export interface Props {
 }
 
 export const ONSPanel = (props: Props) => {
-    const className = "panel panel--" + (props.status ? props.status : "info") + " panel--no-title " + (props.spacious ? "panel--spacious" : "") + " u-mt-m";
+    const className = "ons-panel ons-panel--" + (props.status ? props.status : "info") + " ons-panel--no-title " + (props.spacious ? "ons-panel--spacious" : "") + " u-mt-m";
```

Comment:
I've noticed the `u-mt-*` has been replaced with `ons-u-mt-*` elsewhere, this one appears to have somehow been missed.